x-pack/platform/plugins/shared/cases/server/common/types/case.ts

```diff
@@ -46,6 +46,7 @@ export interface CasePersistedAttributes {
   title: string;
   total_alerts: number;
   total_comments: number;
+  total_events: number;
```

Comment:
It seems you are not introducing usage of this field yet (which makes sense) but I think it would be a bit more accurate/safe to type it as `total_events?: number`. That would prevent other code from relying on the presence of this field. In your next iteration, after the database migration has run, you can treat it as a required field.

I'd consider doing the same for the RT schema.